src/modules/commander/PreflightCheck.cpp

```diff
@@ -570,12 +612,12 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, bool optional, bool report_
 }
 
 bool preflightCheck(orb_advert_t *mavlink_log_pub, bool checkSensors, bool checkAirspeed, bool checkRC, bool checkGNSS,
-		    bool checkDynamic, bool isVTOL, bool reportFailures, bool prearm, hrt_abstime time_since_boot)
+		    bool checkDynamic, bool checkPower, bool isVTOL, bool reportFailures, bool prearm, const hrt_abstime &time_since_boot)
```

Comment:
This list of bools needs to go away in the future, that's super error-prone. A `struct` containing the bools would already be better.

Reply:
Completely agree. I was going to start using vehicle_status_flags and vehicle_status after this PR.